code/modules/supermatter/supermatter.dm

```diff
@@ -71,6 +71,10 @@
 	var/list/message_exclusions = list(/obj/effect/effect/sparks,/obj/effect/overlay/hologram)
 	machine_flags = MULTITOOL_MENU
 
+	light_color = LIGHT_COLOR_ORANGE
```

Comment:
Orange glow?

Previously supermatter didn't glow at all so CL this, also I think yellow would be a good fit

Reply:
Same as a fire, but yeah.